.github/workflows/glow-build.yml

```diff
@@ -0,0 +1,55 @@
+name: build glow
+
+on:
+  pull_request:
+  push:
+    branches:
+      - nightly
+      - main
+      - release/*
+  workflow_dispatch:
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        python_version: ["3.8"]
+      fail-fast: false
+    uses: pytorch/test-infra/.github/workflows/linux_job.yml@main
+    with:
+      runner: linux.12xlarge
+      repository: pytorch/glow
+      timeout: 120
+      script: |
+        # echo '::group::Setup Environment Variables'
+        # Mark Build Directory Safe
+        # git config --global --add safe.directory /__w/audio/audio
+
+        # Set up Environment Variables
+        export PYTHON_VERSION="${{ matrix.python_version }}"
```

Comment:
can hardcode this to 3.8 and delete lines 14-17 from this file

Reply:
Thanks Omkar!
I think glow stack does not care about 3.8 or 3.10. is remaining the old three version python gets it fits all or gets the CI runs 3 times?

I mean, ideally, we should be using whatever default python. So which way is better please?

Reply:
Looks like `build.sh` script is installing Python3.9 from source, so maybe we don't have to specify anything in the `glow-build.yml` file. I guess you can just remove any reference to python version (Line 14-17 and Line 29)